bedrock/legal/templates/legal/emails/fraud-report.txt

```diff
@@ -10,13 +10,13 @@ A new violating website report has been submitted with the following information
 {{ input_product }}
 
 + Specific product
-{{ input_specific_product|safe|strip_tags }}
+{{ input_specific_product|strip_tags|safe }}
```

Comment:
It's *probably* fine in a plain text email , but there have been issues in the past where certain combinations of email client or web mail clients would ignore MIME types. That said, the recipients of these particular emails are highly unlikely to be using ancient versions of outlook express (at least I hope so!), so my knee-jerk reaction was probably a little strong given the context.